File: server/src/schemas/tutorial.py
from typing import List, Optional
from pydantic import BaseModel


class TutorialBase(BaseModel):
    course_id: str
    name: str
    capacity: int
    start_time: str
    end_time: str
    day: str
    location: str = "Online"

    # datetimes: List[str]


class TutorialInfo(TutorialBase):
    id: str
    member_count: int

File: lib/types/api/tutorial.ts
export interface Tutorial {
  id: string;
  name: string;
  capacity: number;
  member_count: number;
  day: string;
  start_time: string;
  end_time: string;
  location: string;
}

export interface TutorialMember {
  id: string;
  first_name: string;
  last_name: string;
  handle: string;
  email: string;
  role: string;
  group: string;
}
